.github/CODEOWNERS

```diff
@@ -148,7 +153,15 @@ Doc/c-api/stable.rst          @encukou
 **/*itertools*                @rhettinger
 **/*collections*              @rhettinger
 **/*random*                   @rhettinger
-**/*queue*                    @rhettinger
+Doc/**/*queue*                @rhettinger
+PCbuild/**/*queue*            @rhettinger
+Modules/_queuemodule.c        @rhettinger
+Lib/*queue*.py                @rhettinger
+Lib/asyncio/*queue*.py        @rhettinger
+Lib/multiprocessing/*queue*.py  @rhettinger
+Lib/test/*queue*.py           @rhettinger
+Lib/test_asyncio/*queue*.py   @rhettinger
+Lib/test_multiprocessing/*queue*.py  @rhettinger
```

Comment:
The last matching pattern in `CODEOWNERS` takes the precedence, so you can do:
```
**/*queue*                    @rhettinger
support/interpreters/queues.py  @ericsnowcurrently
```
to set @rhettinger as owner of all the `*queue*` files except `support/interpreters/queues.py`, which will be owned by @ericsnowcurrently instead.

Moving the subinterpreter section after the `**/*queue*` line should also override the owner for `support/interpreters/queues.py` without further changes.

Reply:
I thought there was something like that but I couldn't find it via web search.  Thanks for explaining.  I'll fix it in a follow-up PR.